### PR TITLE
160 drive 2024 lift motors

### PR DESCRIPTION
**Pull request by gdoffe**

### CODE REVIEW

File: applications/cup2024-robot-motors/Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+APPLICATION = cup2024-robot-motors
+
+BOARD ?= cogip-board
+#BOARD ?= cogip-native
```
Comment (ecourtois) [inline applications/cup2024-robot-motors/Makefile:4]: Remove commented code?

File: platforms/pf-robot-motors/pf_positional_actuators.cpp [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+
+    return 0;
+}
+
+/// Update current speed from quadrature encoders measure.
+static void pf_motor_encoder_read_bottom_lift(double &current_speed)
+{
+    //int32_t cnt = qdec_read_and_reset(MOTOR_BOTTOM_LIFT_ID);
+    //std::cout << cnt << std::endl;
+    //current_speed = (cnt * QDEC_BOTTOM_LIFT_POLARITY) / pulse_per_mm;
```
Comment (ecourtois) [inline platforms/pf-robot-motors/pf_positional_actuators.cpp:201]: Remove commented code?

File: platforms/pf-robot-motors/pf_positional_actuators.cpp [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+
+static void pf_motor_drive_top_lift(const int command, cogip::motion_control::BaseControllerEngine &motor_engine) {
+    pf_motor_drive(command, motor_engine, MOTOR_TOP_LIFT_ID);
+}
+
+void pf_init_motors_sequence(void) {
+    // Move motors until blocked to initialize them.
+    // WARNING: This implies to configure Rsense on motor board correctly.
+    //          Wrong Rsense configuration could damage the motor.
+    //_positional_actuators[Enum::MOTOR_TOP_LIFT]->actuate(150);
```
Comment (ecourtois) [inline platforms/pf-robot-motors/pf_positional_actuators.cpp:260]: Commented code?

**Review by ecourtois — APPROVED** [review]
Just a few commented code that may be removed.
And I'm not the author of any of these files ;-)
CI doesn't pass.
But anyway, I've tested it, so it's fine for me.
You can merge after fixing my remarks.

File: platforms/pf-robot-motors/pf_positional_actuators.cpp (hunk 3 above)
Comment (gdoffe) [reply platforms/pf-robot-motors/pf_positional_actuators.cpp]: done

File: platforms/pf-robot-motors/pf_positional_actuators.cpp (hunk 2 above)
Comment (gdoffe) [reply platforms/pf-robot-motors/pf_positional_actuators.cpp]: done

File: applications/cup2024-robot-motors/Makefile (hunk 1 above)
Comment (gdoffe) [reply applications/cup2024-robot-motors/Makefile]: done

**Comment by gdoffe** [comment]
@ecourtois please have a look at the 2 last commits, added today.

**Review by ecourtois — APPROVED** [review]
LGTM!

**Pull request merged**